scripts/main.js

```diff
@@ -201,4 +201,27 @@ function render() {
   });
 }
 
+const contactForm = document.querySelector('#contact-form');
+const errorSpan = document.querySelector('.error');
+const emailv = document.querySelector('#email');
+
+contactForm.addEventListener('submit', (event) => {
+  event.preventDefault();
+
+  if (emailv.value.trim().match(/[A-Z]/g)) {
+    errorSpan.textContent = 'Email field should not have capital letters.';
+    errorSpan.classList.add('visible');
+    emailv.classList.add('field-error');
+  } else {
+    contactForm.submit();
+  }
+});
```

Comment:
- [ ] Great job here. It will be nice for you to place this inside a function. And use it to submit your form.

Reply:
Hi enaburekhan,
I will take the validation code out in a separate function.
Thank you.